src/directory/directory.js

```diff
@@ -381,7 +381,7 @@ const directory = {
           {
             title: "Other methods",
             route: "/lib/datastore/other-methods",
-            filters: ["android", "ios", "js"],
+            filters: ["android", "flutter", "ios", "js"],
```

Comment:
I am not sure how the filters work, but if all platforms have this section, can this simply be empty?

Reply:
Yes that works, but I decided to follow what exists in the codebase already... listing all possible platforms may also be a bit clearer than an empty array in the case that any new platform could be added in the future.

Reply:
Ah okay. I saw all the empty filters below this in the diff and assumed that generally the filters were excluded if it applied to all, but now I see those are for the CLI. I do see that generally all the platforms are listed in the category docs 👍